app/src/main/java/ch/beerpro/presentation/details/DetailsActivity.java

```diff
@@ -125,6 +122,13 @@ public void showBottomSheetDialog() {
         View view = getLayoutInflater().inflate(R.layout.single_bottom_sheet_dialog, null);
         BottomSheetDialog dialog = new BottomSheetDialog(this);
         dialog.setContentView(view);
+
+        Button ratingButton = (Button)dialog.findViewById(R.id.giveRating);
+        ratingButton.setOnClickListener(v -> {
+            addNewRating(myRatingBar);
+            dialog.dismiss();
+        });
```

Comment:
I would write out the variable names to improve legibility.

Reply:
This is how android automatically creates Lambdas